### ISSUE TITLE
Attribute error in test_network.ExecTestEnv.test_get_gre_tunnel_key

### ISSUE THREAD

**Issue by ssorenso** [F5Networks/f5-openstack-agent#874]
* Title: A short but descriptive summary of the issue, whether it be a bug or enhancement.
* Attachments:
* Details: 

##### Found in test, test type (suggested):
Bug - however, it is a bug in the SDK as far as I can tell... it's a change in functionality if they once allowed for LazyAttributes.

##### Traceback:
```
Running test_vxlan_profile
___________________________ test_get_gre_tunnel_key ____________________________

setup = <class 'test_network.ExecTestEnv'>

    @log_test_call
    def test_get_gre_tunnel_key(setup):
        te = setup
        name = 'test_tunnel'
        profile = 'gre'  # pre-exiting profile on BIGIP
        local_ip = '192.168.102.1'
        remote_ip = '192.168.102.2'
        t = te.mgmt_root.tm.net.tunnels.tunnels.tunnel
        t.create(name=name, partition=te.partition,
                 localAddress=local_ip, remoteAddress=remote_ip, profile=profile)
        add_resource_teardown(te.request, t)
        key = network_helper.get_l2gre_tunnel_key(te.mgmt_root, name, te.partition)
>       assert(key == t.key)

test_network.py:140:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

container = <f5.bigip.tm.net.tunnels.Tunnel object at 0x7f46a46cbd90>
name = 'key'

    def __getattr__(container, name):
        # ensure this object supports lazy attrs.
        cls_name = container.__class__.__name__
        if 'allowed_lazy_attributes' not in container._meta_data:
            error_message = ('"allowed_lazy_attributes" not in',
                             'container._meta_data for class %s' % cls_name)
>           raise LazyAttributesRequired(error_message)
E           LazyAttributesRequired: ('"allowed_lazy_attributes" not in', 'container._meta_data for class Tunnel')

../../../.tox/singlebigip/local/lib/python2.7/site-packages/f5/bigip/mixins.py:95: LazyAttributesRequired
```

##### Tests impacted:
MD5Sum: `69f234e4269cecc6fc844977190a9c67`
* `openstack_agent_liberty_12.1.2-overcloud.6.consoleText`
  * Type: Failure
  * Test: `test_vxlan_profile`
* `openstack_agent_newton_12.1.2-overcloud.6.consoleText`
  * Type: Failure
  * Test: `test_vxlan_profile`

#### OpenStack Release
All supported

#### Agent Version
Lateest

#### Operating System
Nightly

**Comment by dflanigan** [comment]
@ssorenso Can you file an SDK bug and then point to it from here ?

**Comment by richbrowne** [comment]
Fixed by 0cdebc5be80dc4ba8759672f6ee808123c590e5e
